static/js/apps/eval_retrieval_generation/rag_ans_feedback.tsx

```diff
@@ -22,21 +22,42 @@ import { Button } from "reactstrap";
 import { loadSpinner, removeSpinner } from "../../shared/util";
 import { ClaimCounter } from "./claim_counter";
 import {
-  QUERY_FALSE_CLAIMS_FEEDBACK_KEY,
-  QUERY_TOTAL_CLAIMS_FEEDBACK_KEY,
+  QUERY_FALSE_INF_CLAIMS_KEY,
+  QUERY_FALSE_STAT_CLAIMS_KEY,
+  QUERY_TABLES_USED_KEY,
+  QUERY_TOTAL_INF_CLAIMS_KEY,
+  QUERY_TOTAL_STAT_CLAIMS_KEY,
+  QUERY_UNSUB_INF_CLAIMS_KEY,
 } from "./constants";
 import { AppContext, SessionContext } from "./context";
-import { getField, getPath, saveToSheet, setField } from "./data_store";
+import { getAllFields, getPath, saveToSheet, setFields } from "./data_store";
 import { EvalList } from "./eval_list";
 import { FeedbackNavigation } from "./feedback_navigation";
 import { TablePane } from "./table_pane";
 import { EvalType } from "./types";
 
 const LOADING_CONTAINER_ID = "form-container";
+const EMPTY_COUNTS = {
+  [QUERY_TOTAL_STAT_CLAIMS_KEY]: 0,
+  [QUERY_FALSE_STAT_CLAIMS_KEY]: 0,
+  [QUERY_TOTAL_INF_CLAIMS_KEY]: 0,
+  [QUERY_FALSE_INF_CLAIMS_KEY]: 0,
+  [QUERY_UNSUB_INF_CLAIMS_KEY]: 0,
+  [QUERY_TABLES_USED_KEY]: 0,
+};
+const COUNTER_LABELS = {
+  [QUERY_TOTAL_STAT_CLAIMS_KEY]: "Total claims",
+  [QUERY_FALSE_STAT_CLAIMS_KEY]: "False claims",
+  [QUERY_TOTAL_INF_CLAIMS_KEY]: "Total claims",
+  [QUERY_FALSE_INF_CLAIMS_KEY]: "False claims",
+  [QUERY_UNSUB_INF_CLAIMS_KEY]: "Unsubstantiated claims",
+  [QUERY_TABLES_USED_KEY]: "Unique tables used",
+};
```

Comment:
oh, maybe put all of these together with the key list into one object? That's easier to track.

Reply:
good idea, updated